### PR TITLE
test list backend, fe fixes, http clients

### PR DESCRIPTION
**Pull request by Natelai**

### CODE REVIEW

File: testing_platform/Contracts/APICommunication/Enums/TestCategory.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Contracts.APICommunication.Enums;
+
+public enum TestCategory
+{
+    BugReports = 1,
+    TestCases = 2,
+    UnitTests = 3
```
Comment (OlegShado) [inline testing_platform/Contracts/APICommunication/Enums/TestCategory.cs:7]: 2 additional categories are required: AutomatedTesting and CheckLists

File: testing_platform/be/Domain/Enums/TestCategory.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Domain.Enums;
+
+public enum TestCategory
+{
+    BugReports = 1,
+    TestCases = 2,
+    UnitTests = 3
```
Comment (OlegShado) [inline testing_platform/be/Domain/Enums/TestCategory.cs:7]: Similar to problem to testing_platform/Contracts/APICommunication/Enums/TestCategory.cs

File: testing_platform/be/DTOs/Responses/Auth/Tests/TestPreviewDto.cs
Comment (OlegShado) [inline testing_platform/be/DTOs/Responses/Auth/Tests/TestPreviewDto.cs:1]: Author name is required

File: testing_platform/be/Domain/dbo/Test.cs
Comment (OlegShado) [inline testing_platform/be/Domain/dbo/Test.cs:1]: Author name is required

File: testing_platform/Contracts/APICommunication/TestPreviewDto.cs
Comment (OlegShado) [inline testing_platform/Contracts/APICommunication/TestPreviewDto.cs:1]: Author name is required

File: testing_platform/fe/Frontend.Presentation/Pages/MainPage.razor
Comment (OlegShado) [inline testing_platform/fe/Frontend.Presentation/Pages/MainPage.razor:1]: 2 additional categories are required: AutomatedTesting and CheckLists

**Review by OlegShado — APPROVED** [review]

**Review by dariakyselytsia — APPROVED** [review]

**Pull request merged**